data/records/atlas-hi-2024-hi-2015-data.json

```diff
@@ -0,0 +1,513 @@
+        "size": 14214,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287843_file_index.json"
+      },
+      {
+        "checksum": "adler32:ff5a9d25",
+        "size": 14214,
+        "type": "index.txt",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287843_file_index.txt"
+      },
+      {
+        "checksum": "adler32:97b55093",
+        "size": 24863,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287866_file_index.json"
+      },
+      {
+        "checksum": "adler32:97b55093",
+        "size": 24863,
+        "type": "index.txt",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287866_file_index.txt"
+      },
+      {
+        "checksum": "adler32:d9f89348",
+        "size": 9629,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287924_file_index.json"
+      },
+      {
+        "checksum": "adler32:d9f89348",
+        "size": 9629,
+        "type": "index.txt",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287924_file_index.txt"
+      },
+      {
+        "checksum": "adler32:f0b952e0",
+        "size": 29439,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287931_file_index.json"
+      },
+      {
+        "checksum": "adler32:f0b952e0",
+        "size": 29439,
+        "type": "index.txt",
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/rucio/data15_hi/file-indexes/data15_hi_Run_00287931_file_index.txt"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "methodology": {
+      "description": "<p>These data were created during LS2 as part of a major reprocessing campaign of the Run 2 data. All data were reprocessed using Athena Release 21, and new corresponding MC simulation samples were produced. These data and MC simulation datasets were processed into DAOD_HION14 format files; this is a light-weight data format intended for analysis of heavy ion minimum bias data, sufficient to support a wide variety of ATLAS analyses."
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "80036",
+    "relations": [
+      {
+        "description": "For citing all the Heavy Ion Open Data for Research from this release, and to find other related datasets, please see",
+        "doi": "10.7483/OPENDATA.ATLAS.9HK7.P5SI",
+        "recid": "80020",
+        "title": "DAOD_HION14 format 2015 Pb-Pb Open Data for Research from the ATLAS experiment",
+        "type": "isChildOf"
+      }
+    ],
+    "run_period": [
+      "p6480"
```

Comment:
@zlmarshall Please note the `run_period` and the `date_created` value to be "p6480" instead of "2015" or "2016" here...  Something to amend in the data curation scripts?

Reply:
Yep, I agree — done!